tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
+using OmniSharp.Models.WorkspaceInformation;
+using TestUtility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.Script.Tests
+{
+    public class WorkspaceInformationTests : AbstractTestFixture
+    {
+        public WorkspaceInformationTests(ITestOutputHelper output)
+            : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task SingleCsiScript()
+        {
+            using (var testProject = TestAssets.Instance.GetTestScript("SingleCsiScript"))
+            using (var host = CreateOmniSharpHost(testProject.Directory))
+            {
+                var workspaceInfo = await GetWorkspaceInfoAsync(host);
+                var scriptProjects = workspaceInfo.Projects.ToArray();
+                Assert.Single(scriptProjects);
+
+                var project = scriptProjects[0];
+                Assert.Equal("main.csx", Path.GetFileName(project.Path));
+
+                // should have reference to mscorlib
+                Assert.True(project.ImplicitAssemblyReferences.Any(r => r == GetMsCorlibPath()), "Missing reference to mscorlib");
```

Comment:
It feels like maybe this could be extracted to a helper since it's used everywhere?